defaults/main.yml

```diff
@@ -14,6 +14,7 @@ LOAD_BALANCER_CERTS_DIR: /etc/haproxy/certs
 LOAD_BALANCER_CONF_DIR: /etc/haproxy/conf.d
 LOAD_BALANCER_BACKENDS_DIR: /etc/haproxy/backends
 LETSENCRYPT_ARCHIVE_DIR: /etc/letsencrypt/archive
+LETSENCRYPT_LIVE_DIR: /etc/letsencrypt/live
```

Comment:
Oh my apologies!  I was looking at the master versions of those files.

Thank you for the explanation about manage_certs.py/